components/textfield/metadata/textarea.yml

```diff
@@ -8,37 +8,124 @@ sections:
      See the [Textfield migration guide](textfield.html#migrationguide) for migration information.
 examples:
   - id: textfield
-    name: Standard
+    name: Standard Sizes
```

Comment:
Resizing appears not to be possible anymore due to the styles on multiline - [demo page](https://pr-1544--spectrum-css.netlify.app/docs/textarea.html#standardsizes) - is this desirable? How do apps enable resizing?